### PR TITLE
metrics: add new row panel for tidb query

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: ref #49318

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

image

Side effects

Documentation

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by nolouch** [comment]
Can it select a group variable?

**Comment by nolouch** [comment]
and seems here no group result:
image

**Comment by bufferflies** [comment]
> Can it select a group variable?
yes, I have add new select.
image

**Comment by bufferflies** [comment]
> and seems here no group result: image

In this picture, I use the root role to run some ddl, the root doesn't belong any resource group.

File: pkg/metrics/grafana/tidb_resource_control.json [hunk 1]
```diff
@@ -3569,6 +3569,605 @@
...
+              "show": true
+            }
+          ],
+          "yaxis": {
+            "align": false,
+            "alignLevel": null
+          }
+        }
+      ],
+      "title": "Resource Group Details",
```
Comment (nolouch) [inline pkg/metrics/grafana/tidb_resource_control.json:4169]:
```suggestion
      "title": "Query Summary",
```

File: pkg/metrics/grafana/tidb_resource_control.json [hunk 2]
```diff
@@ -3684,6 +4283,33 @@
...
+        "allValue": ".*",
+        "current": {},
+        "datasource": "${DS_TEST-CLUSTER}",
+        "definition": "",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "Resource Group",
+        "multi": false,
```
Comment (nolouch) [inline pkg/metrics/grafana/tidb_resource_control.json:4297]: `true` is better, can choose multiple.

**Review by nolouch — COMMENTED** [review]
rest lgtm

File: pkg/metrics/grafana/tidb_resource_control.json (hunk 1 above)
Comment (bufferflies) [reply pkg/metrics/grafana/tidb_resource_control.json]: fixed

File: pkg/metrics/grafana/tidb_resource_control.json [hunk 3]
```diff
@@ -3569,6 +3569,605 @@
...
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "histogram_quantile(0.999, sum(rate(tidb_server_handle_query_duration_seconds_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\",instance=~\"$tidb_instance\"}[1m])) by (le,resource_group))",
```
[the remark this answers is not in the document]
Comment (bufferflies) [reply pkg/metrics/grafana/tidb_resource_control.json]: fixed

**Review by nolouch — APPROVED** [review]
lgtm

**Review by glorv — APPROVED** [review]
LGTM

**Comment by glorv** [comment]
```
    data_feature_usage_test.go:508: 
        	Error Trace:	pkg/telemetry/data_feature_usage_test.go:508
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTxnSavepointUsageInfo
```
@bufferflies there is a unit test failure.

**Pull request merged**